<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>credit</class>
 <widget class="QDialog" name="credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Credit Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: #434343;
}</string>
  </property>
  <widget class="QLabel" name="label_sum">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>Сумма кредита</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_percent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: #434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>Срок кредита</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_amount">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #DCDCDC;
    border: 1px solid #434343;
	text-align: left;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_term">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #DCDCDC;
    border: 1px solid #434343;
	text-align: left;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_percent">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #DCDCDC;
    border: 1px solid #434343;
	text-align: left;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_annuity">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>205</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
    background-color: #DCDCDC;
    color:  #434343;
    border: 1px solid #434343;
    padding: 5px;
	margin: 5px;
    font-size: 13px;
}

QRadioButton::indicator {
    width: 20px;
    height: 20px;
    border-radius: 10px;
    border: 1px solid  #434343;
}

QRadioButton::indicator:checked {
    background-color: #F6872E;
    border: 1px solid #434343;
}

QRadioButton::indicator:unchecked {
    background-color: #ffffff;
}
</string>
   </property>
   <property name="text">
    <string>Аннуитетные</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_differentiated">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>255</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
    background-color: #DCDCDC;
    color:  #434343;
    border: 1px solid  #434343;
    padding: 5px;
    margin: 5px;
    font-size: 13px;
}

QRadioButton::indicator {
    width: 20px;
    height: 20px;
    border-radius: 10px;
    border: 1px solid  #434343;
}

QRadioButton::indicator:checked {
    background-color: #F6872E;
    border: 1px solid #434343;
}

QRadioButton::indicator:unchecked {
    background-color: #ffffff;
}
</string>
   </property>
   <property name="text">
    <string>Дифференцированные</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>месяцев</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_equal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color:#ffffff;
	font-size: 16px;
	font-weight: bold;
    background-color: #F6872E;
    border: 1px solid #434343;
    padding: 10px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #D17127, stop: 1 #F6872E);
}</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>261</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>pictures/kot1.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_month_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>Ежемесячный платеж</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_overprice_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>Переплата по кредиту</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_full_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>Общая выплата</string>
   </property>
  </widget>
  <widget class="QLabel" name="line_monthly_payment">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #DCDCDC;
    border: 1px solid #434343;
	text-align: left;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="line_overpayment">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #DCDCDC;
    border: 1px solid #434343;
	text-align: left;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="line_total_payout">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>460</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #DCDCDC;
    border: 1px solid #434343;
	text-align: left;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_percent_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: #434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>Тип платежей</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
